src/app/lib/README.md

```diff
@@ -16,13 +16,13 @@ Here are the font sizes and line heights in `em`s, assuming a base font size of
 | :----------- | :------------ | :-------------- | :------------ | :-------------- | :------------ | :-------------- | :--------------------------------------------------------------- |
 | **Type**     | **Font Size** | **Line Height** | **Font Size** | **Line Height** | **Font Size** | **Line Height** | **Article Usage**                                                |
 | Canon        | 1.75          | 2               | 2             | 3.5             | 2.75          | 3               | Headline (h1)                                                    |
-| Trafalgar    | 1.25          | 1.5             | 1.5           | 2.5             | 2             | 2.25            | Subheadline (h2)                                                  |
+| Trafalgar    | 1.25          | 1.5             | 1.5           | 2.5             | 2             | 2.25            | Subheadline (h2)                                                 |
 | Paragon      | 1.25          | 1.5             | 1.375         | 2               | 1.75          | 2               |                                                                  |
 | Double Pica  | 1.25          | 1.5             | 1.25          | 1.875           | 1.5           | 1.75            |                                                                  |
 | Great Primer | 1.125         | 1.375           | 1.125         | 1.5             | 1.25          | 1.5             | Heading for single onward journey box (e.g. More on Archaeology) |
 | Body Copy    | 0.9375        | 1.25            | 1             | 1.5             | 1             | 1.375           | Body copy (p)                                                    |
 | Pica         | 0.9375        | 1.25            | 1             | 1.375           | 1             | 1.25            | Headline for single onward journey                               |
-| Long Primer  | 0.9375        | 1.125           | 0.9375        | 1.125            | 0.875         | 1.125           | Image caption & Video caption & Onward journey timestamps        |
+| Long Primer  | 0.9375        | 1.125           | 0.9375        | 1.125           | 0.875         | 1.125           | Image caption & Video caption & Onward journey timestamps        |
```

Comment:
Minor capitalization inconsistency with line 22

Reply:
I haven't changed this, the editor is removing empty whitespace, but I'll edit any who